string Sender
---
string Model
uint32 CountJoints